src/App.jsx

```diff
@@ -14,57 +16,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+// const classNames = require('classnames');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+const DEFAULT_GOOD = goods.find(good => good === 'Jam');
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+export const App = () => {
+  const [selectedProduct, setSelectedProduct] = useState(DEFAULT_GOOD);
+  const isChosenProduct = good => good === selectedProduct;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1
+        className={classNames('title', {
+          'is-flex is-align-items-center': selectedProduct,
+        })}
+      >
+        {selectedProduct
+          ? `${selectedProduct} is selected`
+          : 'No goods selected'}
+        {selectedProduct && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedProduct('')}
+          />
+        )}
+      </h1>
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            return (
+              <tr
+                data-cy="Good"
+                className={classNames({
+                  'has-background-success-light': isChosenProduct(good),
+                })}
+                key={good}
+              >
+                <td>
+                  <button
+                    data-cy={`${isChosenProduct(good) ? 'RemoveButton' : 'AddButton'}`}
```

Comment:
`isChosenProduct(good)` you do it a few times, just create a variable and use it